### PR TITLE
use new readme theme

### PR DESCRIPTION
**Pull request by amanmdesai**
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to respond to your contribution.
To ensure that the right people get notified, you can tag some of the last contributors with `@githubname`.

Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum. If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact the HSF training convenors ([contacts here](https://hepsoftwarefoundation.org/workinggroups/training.html)).

You may delete these instructions from your pull request.

\- HSF Training
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,95 @@
...
+repository. For making your contribution, we use the GitHub flow, which is
+nicely explained in the chapter [Contributing to a Project][progit] in Pro Git
+by Scott Chacon.
+Look for the tag [![good_first_issue]][gfi-badge], which marks particularly simple issues to get you started.
 
-Current maintainers of this lesson are
+<!-- END CENTRALLY MAINTAINED SECTION -->
+## 💖 Authors
 
-* Anna Scaife
```
Comment (klieret) [inline README.md:29]: we should keep mentioning Anna until we properly add all the authors

**Review by klieret — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (amanmdesai) [reply README.md:29]: @klieret  thanks for the comment. I have updated the readme.

**Comment by amanmdesai** [comment]
This will close #15

File: README.md [hunk 2]
```diff
@@ -1,40 +1,97 @@
...
+the content of the lesson:
 
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
 
-## Authors
+* Anna Scaife
```
Comment (klieret) [inline README.md]: that should be above the 'all contributors' place holder, else it will be overwritten

**Review by klieret — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (amanmdesai) [reply README.md]: I updated that, thanks

**Review by klieret — APPROVED** [review]
Thank you!

**Pull request merged**